docs/src/modules/components/TopLayoutBlog.js

```diff
@@ -397,7 +400,20 @@ export default function TopLayoutBlog(props) {
             </React.Fragment>
           ) : null}
           {rendered.map((chunk, index) => {
-            return <MarkdownElement key={index} renderedMarkdown={chunk} />;
+            return (
+              <RichMarkdownElement
+                key={index}
+                demos={demos}
```

Comment:
I doesn't seem that should allow the blog to render demos. See my comment above for why

Reply:
> clunky and wonky

Yes, but it's a lot more likely to keep working in the long term without slowing us down (breaking changes).